### PR TITLE
split update_images file

### LINKED ISSUE #29: Housekeeping: Split update_images and test_update_images
The catch all *update_images files are getting too long. Let's split them up and introduce some structure into the project.

### PR DESCRIPTION
**Pull request by miyunari**
Now we have method calls like `aws.get_aws_regions`, would it make sense to rename it in `get_regions` as it is coming from the `aws` file? 

closes #29

### CODE REVIEW

**Comment by FKolwa** [comment]
> Now we have method calls like `aws.get_aws_regions`, would it make sense to rename it in `get_regions` as it is coming from the `aws` file?
> 
> closes #29

Yeah I agree, with your split into submodules we can avoid name redundancies. Feel free to rename them.

**Comment by miyunari** [comment]
Great, then I will wait until #37 got merged. Adapt this PR accordingly and change all names afterwards in an extra PR? :)

**Review by FKolwa — APPROVED** [review]
LGTM!

**Pull request merged**
